src/handles/handleFormat.ts

```diff
@@ -48,7 +61,7 @@ async function handleFormat(
       output += stderr
     }
   }
-  return await next(output)
+  return next(output)
```

Comment:
See https://eslint.org/docs/rules/no-return-await

Reply:
Thanks for the notice